realsense2_description/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(realsense2_description)
+
+find_package(catkin REQUIRED COMPONENTS
+    )
+
+# RealSense description
+catkin_package(
+    )
+
+# Install launch files
+install(DIRECTORY launch/
+    DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/launch
+    )
```

Comment:
All of these directories could be installed in a single `install(..)` block:

```cmake
install(DIRECTORY launch meshes rviz urdf
  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
```

Reply:
:+1: Done.